src/components/DocumentProducer/DeliverableEditImages/index.tsx

```diff
@@ -49,9 +49,6 @@ const DeliverableEditImages = (props: DeliverableEditImagesProps): JSX.Element =
 
   return (
     <Grid container spacing={3} sx={{ padding: 0 }} textAlign='left'>
-      <Grid item xs={12}>
-        <Textfield label='' type='text' id='description' value={variable.description} display={true} />
-      </Grid>
```

Comment:
This description `Textfield` is a duplicate as one is already defined in `DeliverableVariableDetailsInput`.